packages/blockchain-wallet-v4-frontend/src/components/Cartridge/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
```

Comment:
Actually we import both new and old component packages so you can use https://blockchain.github.io/components/?selectedKind=Molecules%2FCartridges%2FCartridge&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

Reply:
hey good to know, I did not know the library was supported already. Will do the change thanks !

Reply:
Replaced it, I had to include in the ThemeProvider the palette for the @blockchain-com/components that was missing